docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -4202,6 +4202,37 @@ Time.at(timestamp).in_time_zone
 * https://rails.rubystyle.guide#time
 * http://danilenko.org/2012/7/6/rails_timezones
 
+== Rails/TransactionRequiresNew
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Enabled
+| Yes
+| Yes
+| 2.10
+| -
+|===
+
+
+
+=== Examples
+
+==== Enabled (default)
+
+[source,ruby]
+----
+# bad
+ActiveRecord::Base.transaction do
+  some_database_stuff
+end
+
+# good
+ActiveRecord::Base.transaction(requires_new: true) do
```

Comment:
Is it okay not to specify `joinable: false` together?

Reply:
My understanding is that they work independently.  `requires_new: true` is for custom tranaction (`transaction do ... end`) and `joinable: false` is for the transactions created by `ActiveRecord` methods such as `update!`

Anyways, I will do some research and testing on this.  I have read many blogs/articles about it so far but none of them seem super clear on this topic.  I will try to figure it out and come up with some good documentation and examples.  I might even write a medium article :joy: